stock_vertical_lift/models/vertical_lift_operation_inventory.py

```diff
@@ -142,44 +124,44 @@ def _compute_tray_data(self):
             record.tray_y = location.posy
             record.tray_matrix = location.tray_matrix
 
-    @api.depends("current_inventory_line_id.product_id.packaging_ids")
+    @api.depends("quant_id.product_id.packaging_ids")
     def _compute_product_packagings(self):
         for record in self:
-            product = record.current_inventory_line_id.product_id
+            product = record.quant_id.product_id
             if not product:
                 record.product_packagings = ""
                 continue
             content = self._render_product_packagings(product)
             record.product_packagings = content
 
-    @api.depends("tray_location_id", "current_inventory_line_id.product_id")
+    @api.depends("tray_location_id", "quant_id.product_id")
     def _compute_tray_qty(self):
         for record in self:
-            if not (record.tray_location_id and record.current_inventory_line_id):
+            if not (record.tray_location_id and record.quant_id):
                 record.tray_qty = 0.0
                 continue
-            product = record.current_inventory_line_id.product_id
+            product = record.quant_id.product_id
             location = record.tray_location_id
             record.tray_qty = self._get_tray_qty(product, location)
 
     def _compute_number_of_ops(self):
         for record in self:
-            line_model = self.env["stock.inventory.line"]
+            line_model = self.env["stock.quant"]
             record.number_of_ops = line_model.search_count(
```

Comment:
I would suggest to rename to `quant_model` if it's not an inventory line anymore.